ee/ucp/admin/configure/restrict-services-to-worker-nodes.md

```diff
@@ -1,40 +1,37 @@
 ---
 title: Restrict services to worker nodes
-description: Learn how to configure Universal Control Plane to only allow running services in worker nodes.
+description: Learn how to configure Docker Universal Control Plane to only allow running services in worker nodes.
 keywords: ucp, configuration, worker
 ---
 
 >{% include enterprise_label_shortform.md %}
 
-You can configure UCP to allow users to deploy and run services only in
-worker nodes. This ensures all cluster management functionality stays
-performant, and makes the cluster more secure.
+Docker Universal Control Plane (UCP) is set to run on both manager and worker nodes by default, however, it can be configured to run only on worker nodes. This ensures that all cluster management functionality stays performant, and makes the cluster more secure.
 
-If a user deploys a malicious service that can affect the node where it
-is running, it won't be able to affect other nodes in the cluster, or
-any cluster management functionality.
+If a user deploys a malicious service that can affect the node where it is running, it cannot affect other nodes in the cluster, or any cluster management functionality.
 
-## Swarm Workloads
+## Swarm workloads
 
-To restrict users from deploying to manager nodes, log in with administrator
-credentials to the UCP web interface, navigate to the **Admin Settings**
-page, and choose **Scheduler**.
+To change user options for deploying workloads to manager nodes:
 
-![](../../images/restrict-services-to-worker-nodes-1.png){: .with-border}
+1. Log in to the UCP UI with administrator credentials.
+2. Navigate to **Admin Settings**.
+3. Click **Scheduler**.
```

Comment:
Suggest editing to read: "3. Click **Scheduler** from the left-hand menu. The **{{Insert Dialog/Page Title Here}}** will display."